docs/kerala/getting-started.md

```diff
@@ -0,0 +1,314 @@
+```
+
+ Refer to [using AWS credential files](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) for more information.
+
+* Create an S3 bucket named `simple-server-terraform-state`.
+
+* Create a [DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-1.html) table
+  called `terraform-lock` with `LockID` as primary key.
+
+### 2. Add the new environment to the repository
+
+Run the following script from the `terraform/` directory of the `deployment` repository to create a directory for your
+new environment.
+
+```bash
+$ cd deployment
+$ cd terraform
+$ ./create_aws_account kerala
+```
+
+This will create a `bangladesh/` directory with several files.
+
+* `main.tf`: The main file containing a starting terraform configuration. It includes a demo and production Simple
+  environment for your new account.
+* `terraform.tfvars`: A git-ignored file where database credentials and other sensitive information is stored.
+* `certificate.pem`: Placeholder for your SSL certificate
+* `certificate.chain.pem`: Placeholder for your SSL certificate chain
+* `certificate.private_key.pem`: Placeholder for your SSL certificate private key
+
+### 3. Configure the new environment
+
+You will have to edit these files as follows:
+
+* `main.tf`: Update the `profile` value throughout this file from `sample` to `kerala`. Remove the
+  `module "simple_server_sample_demo"` block if you do not wish to set up a demo environment.
+* `terraform.tfvars`: Choose any username and password for your databases and enter them into this file.
+* `certificate.pem`: Place your SSL certificate in this file.
+* `certificate.chain.pem`: Place your SSL certificate chain in this file.
+* `certificate.private_key.pem`: Place your SSL certificate private key in this file.
+
+### 4. Choose a vault password
+
+### Vault password
+
+Sensitive terraform configuration is stored and checked into Github as encrypted files using
+[ansible-vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html). These files are decrypted locally and
+git-ignored for local development. In order to decrypt and re-encrypt these files, you will need to use a vault password
+file.
+
+Choose your own vault password when you get started. This can be any string. We recommend a 256 character password containing letters, numbers, and special characters.
+
+Create a file `~/.vault_password` on your developer environment and place the password into this file.
+
+See [Ansible documentation](https://docs.ansible.com/ansible/latest/user_guide/vault.html) for more information on how to create and manage vault passwords.
+
+### 5. Encrypt secrets
+
+Run the following script from the `terraform` directory to encrypt the sensitive `tfvars` and `pem` files so they can
+safely be checked into the repository.
+
+```bash
+$ cd terraform
+$ ./encrypt ~/.vault_password kerala
+```
+
+### 6. Add the master AWS SSH key to your machine
```

Comment:
They probably won't have a master key already, we can change this to `6. Create an SSH key`